modules/games.py

```diff
@@ -149,6 +150,38 @@ def addPoints(self, message: core.BotMessage) -> None:
 
         return message.respond("Points added!")
 
+    def removePlayers(self, message: core.BotMessage) -> None:
+        """Remove users from the minigame leaderboard
+
+        Arguments:
+            message {Message} -- the Message object that invoked the command
+        """
+        if not message.room: return message.respond("You can only remove players in a room.")
+        if not message.sender.can("hostgame", message.room): return message.respond("Permission denied.")
+
+        if len(message.arguments) < 2:
+            return message.respond(
+                f"Usage: ``{config.commandCharacter}removeplayers [comma-separated list of users]``."
+            )
+        usernames = message.arguments[1:]
+        usersOnLB = []
+        usersNotOnLB = []
+if message.room.id not in self.minigamePoints.keys(): return message.respond("There are no scores.")
```

Comment:
```suggestion
        if message.room.id not in self.minigamePoints.keys(): return message.respond("There are no scores.")
```

Reply:
I did some tests on a local server and this line results in a websocket error if done in the bot's PM

Reply:
uhhh, that shouldn't be happening? Maybe my websocket library is broken; I wrote it when I was a much worse programmer than I am now.

Reply:
In any case, you should leave the line in - if responding to PMs is broken, that's going to cause issues regardless of this PR.

Reply:
well, anyway, the room.id worked